### DISCUSSION TITLE
RiMusic to YT music

### DISCUSSION THREAD

**Discussion by kris20030705** [fast4x/RiMusic#2232] [General]
Does anyone know how to import ri music playlist to yt music?? Or Spotify?

**Comment by Abilomp** [comment]
What the use of yt music if you have Ri music..after getting this app I
deleted all my music streaming app

**Reply by kris20030705** [reply]
I am trying other music streaming app not really(yt music and Spotify) but an alternative of that..!simp music is same as yt music and spotube is same as Spotify..spotubeis also available for pc

**Comment by fast4x** [comment]
Not there isn't a method.
